packages/common/src/search/_searchContent.ts

```diff
@@ -21,78 +27,101 @@ import { IHubContent, IModel, itemToContent, setContentSiteUrls } from "..";
 export async function _searchContent(
   filter: Filter<"content">,
   options: IHubSearchOptions
-): Promise<IContentSearchResult> {
+): Promise<ISearchResponse<IHubContent>> {
   // expand filter so we can serialize to either api
   const expanded = expandContentFilter(filter);
 
   // APIs
-  if (!options.apis) {
+  if (!options.api) {
     // default to AGO PROD
-    options.apis = ["arcgis"];
+    options.api = "arcgis";
   }
-  const apis = expandApis(options.apis);
 
+  const api = expandApi(options.api);
+
+  let searchPromise;
   // map over the apis, depending on the type we issue the queries
-  const searchPromises = apis.map((api) => {
-    // Portal Search
-    if (api.type === "arcgis") {
-      // serialize for portal
-      const so = serializeContentFilterForPortal(expanded);
-      // pass auth forward
-      if (options.authentication) {
-        so.authentication = options.authentication;
-      }
-      // Aggregations
-      if (options.aggregations?.length) {
-        so.countFields = options.aggregations.join(",");
-        so.countSize = 200;
-      }
-      // copy over various options
-      // TODO: Dry this up
-      if (options.num) {
-        so.num = options.num;
-      }
-      if (options.sortField) {
-        so.sortField = options.sortField;
-      }
-      if (options.sortOrder) {
-        so.sortOrder = options.sortOrder;
-      }
-      // NOTE: I think we will end up setting
-      // the site URL in mergeSearchResults() instead
-      // since it applies to both items and datasets
-      return searchPortal(so, options.site);
+  // const searchPromises = apis.map((api) => {
```

Comment:
After discussions w Andrew, we decided to remove the multi-api complexity. If needed, we can have a higher-level fn that handles dispatching multiple calls to searchContent, and then merging the responses on it's own.